app/js/components/rule/ruleSummary/ruleSummary.jade

```diff
@@ -12,12 +12,4 @@ card.card-rule-summary(expandable='true' init-collapsed='initCollapsed' on-toggl
       .rule-reason(md-flex-100)
       .rule-resolution(md-flex-100)
       .recommended-kbase(ng-if='report.rule.node_id' node='report.rule.node_id')
-      .text-center.border-top(ng-show='report.rule.more_info')
```

Comment:
is "Show more info" removed intentionally?

Reply:
nah O_o